src/github.com/getlantern/balancer/balancer.go

```diff
@@ -22,21 +22,30 @@ var (
 // Balancer balances connections established by one or more Dialers.
 type Balancer struct {
 	dialers []*dialer
+	trusted []*dialer
 }
 
 // New creates a new Balancer using the supplied Dialers.
 func New(dialers ...*Dialer) *Balancer {
```

Comment:
I changed this to create the sorted list of trusted dialers after the list of all dialers.